### PR TITLE
Feat/add flake8

### LINKED ISSUE #26: Add flake8

### PR DESCRIPTION
**Pull request by Melanie-M**
Branched of feat/version-1
Close #26

### CODE REVIEW

File: gnomic/genotype.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def changes(self):
 class Genotype(object):
     def __init__(self, changes, parent=None):
         if parent:
-            state = GenotypeState(parent.state.changes)
```
Comment (lyschoening) [inline gnomic/genotype.py]: Why is this?

Comment (Melanie-M) [reply gnomic/genotype.py]: `GenotypeState` is expecting a list of tuple like `[(before, after), (before, after)]` to initialize `self._changes`

`self.state.changes` is a list of change like `[Change(before, after), ...]`

Comment (lyschoening) [reply gnomic/genotype.py]: It actually expects Change objects. The bug here was that it's not transforming them:

```
class GenotypeState(object):
    _changes: List[Tuple[Union[Annotation, AtLocus, None], Union[Annotation, None]]]
    def __init__(self, changes: Sequence[Change] = ()) -> None:
        ...
```

`GenotypeChange._changes` is private.

Comment (lyschoening) [reply gnomic/genotype.py]: Fixed

File: gnomic/__init__.py [hunk 2]
```diff
@@ -1,279 +1 @@
...
-            return genotype_to_string(self, fusions=fusions)
-        elif output == 'html':
-            return genotype_to_html(self, fusions=fusions)
-        else:
-            raise NotImplementedError('Unknown output format: {}'.format(output))
-
-    def __repr__(self):
-        return '{}({})'.format(self.__class__.__name__, repr({'parent': self.parent, 'changes': self._changes}))
\ No newline at end of file
+from .genotype import Genotype  # noqa
```
Comment (lyschoening) [inline gnomic/__init__.py]: The import statement exists to make it easier to access Genotype. To signal that it is there to be exported and at the same time quiet flake8, specify it in `__all__`:

```
from .genotype import Genotype

__all__ = ('Genotype',)
```

**Review by lyschoening — APPROVED** [review]

**Comment by lyschoening** [comment]
Don't merge into `master` yet!

**Pull request closed without merging**